cluster/src/test/scala/org/apache/spark/sql/SnappySQLQuerySuite.scala

```diff
@@ -253,10 +253,235 @@ class SnappySQLQuerySuite extends SnappyFunSuite {
         "(exists (select col1 from r2 where r2.col1=r1.col1) " +
         "or exists(select col1 from r3 where r3.col1=r1.col1))")
 
-    val result = df.collect()
+    df.collect()
```

Comment:
I had moved the test from ViewTest because it was failing when run as part of core (with upstream Spark -- see SNAP-2342), but now see that Asif already moved it to BugTest in https://github.com/SnappyDataInc/snappydata/commit/c8887c66598cdf5861e3db5098085f9f7a3f10bc so will remove from here.